### ISSUE TITLE
Failed to start the server: Error: Can not load `src/iconfont.ttf?t=1709607579696`

### ISSUE THREAD

**Issue by 289420843** [farm-fe/farm#1142]
### Steps to reproduce

修改版本号 "@farmfe/core": "1.0.12"，错误消失

### What is expected?

。

### What is actually happening?

。

### System Info

```Shell
System:
    OS: Windows 10 10.0.18363
    CPU: (6) x64 Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz
    Memory: 11.91 GB / 31.93 GB
  Binaries:
    Node: 16.18.1 - d:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.19.2 - d:\Program Files\nodejs\npm.CMD
    pnpm: 8.15.3 - d:\Program Files\nodejs\pnpm.CMD
  Browsers:
    Edge: Spartan (44.18362.449.0)
    Internet Explorer: 11.0.18362.1
```

**Comment by 289420843** [comment]
iconfont.css 文件
`@font-face {
  font-family: "iconfont";
  src: url('iconfont.ttf?t=1709607579696') format('truetype');
}
...`

**Comment by wre232114** [comment]
I'll fix it today.

**Comment by wre232114** [comment]
fixed in `@farmfe/core@1.0.16`
